### DISCUSSION TITLE
Build API docs

### DISCUSSION THREAD

**Discussion by hydroper** [cabinpkg/cabin#755] [Ideas]
Did you think of something like `cargo doc`? That will build API docs locally.

Also, crates.io automatically builds docs in a Docker instance and shares its link at the package repository. But I think this is complex, so supporting `poac doc` is enough.

**Comment by ken-matsui** [comment]
@hydroper Yes, that feature is definitely on the roadmap, although its priority is low. Most features of Cargo will be covered eventually.
